runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java

```diff
@@ -634,5 +636,29 @@ private void resetEndpoints()
 	{
 		// Select the first tab (NORMAL hiscores)
 		tabGroup.select(tabGroup.getTab(0));
+
+		// Selects the DMM tab if the current world is DMM.
+		selectWorldEndpoint();
+	}
+
+	private void selectWorldEndpoint()
```

Comment:
This should return a `HiscoreEndpoint` and not have side effects.

Reply:
Changes have been commited